src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -218,9 +219,11 @@ public void Should_Set_Commandline_Switches()
                 fixture.Settings.OutputFile = "stdout.txt";
                 fixture.Settings.ErrorOutputFile = "stderr.txt";
                 fixture.Settings.Full = true;
-                fixture.Settings.Results = "NewTestResult.xml";
-                fixture.Settings.ResultFormat = "nunit2";
-                fixture.Settings.ResultTransform = "nunit2.xslt";
+                fixture.Settings.Results = new[]
+                {
+                    new NUnit3Result { FileName = "NewTestResult.xml", Format = "nunit2", Transform = "nunit2.xslt" },
```

Comment:
@Roemer I'd say it's out of scope for this PR. We should raise another issue to track that (NUnit will crash if both are supplied https://github.com/nunit/nunit-console/issues/252).

Are you interested in raising issue and submitting a PR for that? Note, the PR should be submitted after this is merged to avoid merge conflicts.